test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/RequestTests.cs

```diff
@@ -34,7 +33,6 @@ namespace Microsoft.AspNetCore.Server.Kestrel.FunctionalTests
     public class RequestTests
     {
         private const int _connectionStartedEventId = 1;
-        private const int _connectionKeepAliveEventId = 9;
```

Comment:
On the flip side this makes it slightly more obvious when we make a breaking change to logging by changing the event ids for the events that are expected on a given test.